### PR TITLE
Added New Methods

### PR DESCRIPTION
**Pull request by GameoCoder**
-Added support for .webm downloads
-Added, Convert YT Video to mp3

### CODE REVIEW

File: download_video.py [hunk 1]
```diff
@@ -27,6 +65,32 @@ def download_youtube_video(url):
     except Exception as e:
         print(f"An error occurred: {e}")
 
+def advanced_features():
+    print("Methods Available:-\n     1) Download Youtube Video as Music\n")
+    userChoice = input("Your Choice: ")
+    if(userChoice == "1"):
+        music_url = input("Enter the Youtube Video to be converted to music: ")
+        download_youtube_music(music_url)
+    #elif(userChoice == "2"):
```
Comment (AswinOP) [inline download_video.py:74]: is this commented out  intentionally ?

**Comment by GameoCoder** [comment]
Previously, the youtube music links would also be downloaded as mp4.
This new code will check for wether the url is youtube music, or it is youtube video, and download the file accordingly.
Support for more Features has been added and commented under the function "advanced_features()".

**Comment by GameoCoder** [comment]
To clarify,
Advanced features may include support for
- Download small sections of the video/music
- Using sponsorblock api to download video/music
- Download whole play lists of music and videos
- etc.

**Review by AswinOP — APPROVED** [review]
LGTM!

File: download_video.py (hunk 1 above)
Comment (GameoCoder) [reply download_video.py:74]: Yes, it is for some additional features I would be adding to the downloader. 👍

Comment (GameoCoder) [reply download_video.py:74]: Yes, I thought of adding some additional features that yt-dlp supports.👍

**Pull request merged**